python3.7/cookiecutter-aws-sam-eventbridge-schema-app-python/{{cookiecutter.project_name}}/conftest.py

```diff
@@ -0,0 +1,4 @@
+import sys, os
+
+here = os.path.abspath("{{ cookiecutter.function_name }}")
```

Comment:
This is `here = os.path.abspath("hello_world_function")` in 3.6 example. Which one is right?

Reply:
For the shema-app-python (dynamic) templates, it should be the dynamic variable one For the eventBridge-python templates (static) templates it should be  hello_world_function

if you Ctrl-F for abspath you'll see 6 uses, 3 dynamic, 3 static

The static templats don't have cookiecutter.function_name in teh cookiecutter.json